packages/web/package.json

```
@@ -10,33 +10,40 @@
   "license": "MIT",
   "author": "Motif",
   "type": "module",
-  "exports": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./init": "./dist/init.js"
+  },
   "main": "dist/index.js",
   "files": [
-    "README.md",
-    "dist"
+    "dist/index.*",
+    "dist/init.*"
   ],
   "scripts": {
-    "build": "tsc --build --clean && tsc --build",
-    "dev": "tsc --build --clean && tsc --build --watch",
-    "prepack": "tsc --build --clean && tsc --build",
-    "vite": "vite"
+    "analyze": "node scripts/analyze.js",
+    "build": "node scripts/build.js",
+    "dev": "node scripts/dev.js",
+    "prepack": "node scripts/build.js"
   },
   "dependencies": {
-    "@markprompt/core": "0.4.5",
-    "lit": "^2.7.2",
-    "rehype-sanitize": "^5.0.1",
-    "rehype-stringify": "^9.0.3",
-    "remark-gfm": "^3.0.1",
-    "remark-parse": "^10.0.1",
-    "remark-rehype": "^10.1.0",
-    "unified": "^10.1.2"
+    "@markprompt/core": "^0.4.5"
```

So far we use exact pinned versions.

```suggestion
    "@markprompt/core": "0.4.5"
```

I’m not saying we either should or shouldn’t, but it might be a good idea until we get to a stable release.